vars.tf

```
@@ -314,3 +314,101 @@ variable "github_repository" {
   type        = string
   default     = ""
 }
+
+/*
+ * Variables to support database and S3 backups
+ */
+
+variable "b2_application_key_id" {
+  description = "Backblaze application key ID"
+  type        = string
+  default     = ""
+}
+
+variable "b2_application_key" {
+  description = "Backblaze application key secret"
+  type        = string
+  default     = ""
+}
+
+variable "b2_bucket" {
+  description = "Name of the Backblaze B2 bucket, where secondary backups are sent (if set)"
+  type        = string
+  default     = ""
+}
+
+variable "b2_host" {
+  description = "Name of the Backblaze B2 host, where secondary backups are sent (when b2_bucket is set)"
+  type        = string
+  default     = ""
+}
+
+variable "backup_memory" {
+  default = "32"
```

It may be worth documenting here that this controls both the back-up-db task and the clone-s3-to-b2 task (if that coupling is worth maintaining, for the sake of fewer variables, which seems fine at this stage).
```suggestion
variable "backup_memory" {
  description = "The memory to allocate for each of the backup processes."
  default     = "32"
```